cli/src/main/java/org/opentosca/toscana/cli/commands/transformation/AbstractTransformation.java

```diff
@@ -11,78 +12,93 @@
-@Command(parameterListHeading = "%nParameters:%n",
+@Command(//parameterListHeading = "%nParameters:%n",
     optionListHeading = "%nOptions:%n",
     commandListHeading = "%nCommands:%n")
-public abstract class AbstractTransformation {
+public abstract class AbstractTransformation extends AbstractApiCall {
 
-    @Option(names = {"-p", "--platform"}, paramLabel = Constants.PARAM_PLATFORM, description = "Platform for Transformation Start")
-    private String platform = "";
+    private ApiController api;
+    @Option(names = {"-p", "--platform"}, paramLabel = Constants.PARAM_PLATFORM, description = "Platform for Transformation")
+    private String platform;
     @Option(names = {"-t", "--transformation"}, paramLabel = "<CSAR/Platform>", description = "CSAR and Platform for the Transformation, Slash not allowed in the inputs. Correct format: -t csar/platform")
-    private String transformation = "";
-    @Option(names = {"-c", "--csar"}, paramLabel = Constants.PARAM_CSAR, description = "CSAR for Transformation Start")
-    private String csar = "";
-    @Option(names = {"-v", "--verbose"}, description = "Enable Info Level Process Output")
-    private boolean showVerbose;
-    @Option(names = {"-m", "--moreverbose"}, description = "Enable Debug Level Process Output")
-    private boolean showMVerbose;
+    private String transformation;
+    @Option(names = {"-c", "--csar"}, paramLabel = Constants.PARAM_CSAR, description = "CSAR for Transformation")
+    private String csar;
 
-    private ApiController api;
-    private Constants con;
-
-    public ApiController startApi() {
-        if (showMVerbose) {
-            api = new ApiController(true, false);
-            return api;
-        } else if (showVerbose) {
-            api = new ApiController(false, true);
-            return api;
-        } else {
-            api = new ApiController(false, false);
-            return api;
+    AbstractTransformation() {
+        api = getApi();
+    }
+
+    protected abstract String performCall(ApiController ap, String[] ent);
+
+    @Override
+    public void run() {
+        try {
+            System.out.println(callApi());
+        } catch (IOException e) {
+            System.err.println(e.getMessage());
         }
     }
 
     /**
-     Converts the transformation string into two separate strings that are used to call
-     methods
+     Converts the transformation string into two separate strings that are used to call methods
 
      @param transformation provided Transformation String containing, CSAR and Platform
-     @return the two separated String for CSAR and Platform
-     @throws IOException if the transformation string contains more than one slash,
-     only as separator allowed
+     @return the two separated String for CSAR and Platform only as separator allowed
      */
     private String[] inputTransformation(String transformation) throws IOException {
+        String[] input;
         if (transformation.matches(".*/.*/.*")) {
```

Comment:
This is not a valid regular expression. Replace it with `".*\/.*\/.*"`

Reply:
are you sure? prints illegal escape character

Reply:
Gonna have a look later.

Reply:
did you try two backslashes like this: 
`"\\."`